socorro/external/rabbitmq/crashstorage.py

```diff
@@ -39,8 +35,9 @@ class RabbitMQCrashStorage(CrashStorageBase):
         'rabbitmq_class',
         # we choose a pooled connection because we need thread safe connection
         # behaviors
-        default=ConnectionContextPooled,
+        default='socorro.external.rabbitmq.connection_context.ConnectionContextPooled',
```

Comment:
I changed this to a Python module string that gets converted with the `class_converter` which is how we do all the other classes like this one.